design/ui/pagination/src/Pagination.js

```diff
@@ -0,0 +1,292 @@
+import React, { Fragment } from 'react';
+import pt from 'prop-types';
+import classNames from 'classnames/bind';
+import { noop } from 'lodash';
+
+import { Button, Select, Header, Popup } from 'semantic-ui-react';
+import 'semantic-ui-css-custom';
+import styles from './Pagination.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /** Can render as a different tag or component */
+  as: pt.oneOfType([pt.string, pt.node]),
+  /** the items collection length */
+  count: pt.number,
+  /** entity name in plural form */
+  entityNamePlural: pt.string,
+  /** entity name in singular form */
+  entityNameSingular: pt.string,
+  /** specifies if the data is currently loading */
+  isLoading: pt.bool,
+  /** items per page limit */
+  limit: pt.number,
+  /** a message to be displayed when no items to display */
+  notFoundMessage: pt.string,
+  /** current page */
+  page: pt.number,
```

Comment:
Semantics here also. I think this makes more sense to call this `currentPage` so it is easy to understand when consuming and viewing these components in code. What do you think?

Reply:
absolutely, working on it.